frontend/layouts/Default.vue

```diff
@@ -46,9 +46,9 @@
                   </button>
                   <div class="origin-top-right absolute right-0 mt-2 w-48 rounded-md shadow-lg"
                        v-show="openSupport" @click="toggleSupport()">
-                    <div class="py-1 rounded-md bg-white shadow-xs" role="menu" aria-orientation="vertical"
+                    <div class="py-1 rounded-md bg-white ring-1 ring-black ring-opacity-5" role="menu" aria-orientation="vertical"
                          aria-labelledby="user-menu">
-                      <a href="https://ticketvise.com/getting-started/" target="_blank"
+                      <a href="https://ticketvise.com/getting-started/" target="_blank" rel="noopener"
```

Comment:
Waarom is hier een rel="noopener" toegevoegd?

Reply:
Security reasons, see: [https://web.dev/external-anchors-use-rel-noopener/](https://web.dev/external-anchors-use-rel-noopener/)